omod/src/main/webapp/resources/js/aijar.js

```diff
@@ -35,6 +35,12 @@ jq(document).ready(function () {
             phone_number.match(/^[0-9]{1,10}$/);
     }, "Please specify a valid mobile number without any spaces like 0712345678");
 
+    jq.validator.addMethod("nationalid", function(nationalid, element) {
+        nationalid = nationalid.replace(/\(|\)|\s+|-/g, "");
+        return this.optional(element) || nationalid.match(/^$|^[A-Z][FM]\d{5}([A-Z0-9]){7}$/);
+    }, "Enter a valid National ID example XX11112222XYX5");
```

Comment:
@slubwama National ID Example: use CF12345678ABCD (the duplicated numbers are hard to make sense of)